sns_sqs.tf

```diff
@@ -1,10 +1,18 @@
 ##### Source SQS queue for ESF
-resource "aws_sqs_queue" "esf-queue" {
-  name                       = var.esf-sqs-queue-name
-  visibility_timeout_seconds = 900
-}
 
+data "aws_sqs_queue" "esf-queue" {
```

Comment:
we don't need to create the queues provided from the variable. they will be already existing in the user environment

Reply:
basically we will drop this terraform file